packaging/fedora/snapd.spec

```diff
@@ -112,24 +109,24 @@ ExclusiveArch:  %{ix86} x86_64 %{arm} aarch64 ppc64le s390x
 %endif
 
 # If go_compiler is not set to 1, there is no virtual provide. Use golang instead.
-BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang} >= 1.9
+BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang >= 1.9}
```

Comment:
I didn't expect `%{foo > 123}` to work, is this intentional?

Reply:
It doesn't. It's `%{!?go_compiler:golang >= 1.9}`. That means that if `%go_compiler` is not defined, it uses `golang >= 1.9`.